code/datums/storage/storage.dm

```diff
@@ -1075,13 +1017,23 @@ GLOBAL_LIST_EMPTY(cached_storage_typecaches)
 		var/atom/movable/movable_loc = real_location
 		movable_loc.become_active_storage(src)
 
-	orient_to_hud()
+	var/ui_style = ui_style2icon(toshow.client?.prefs?.read_preference(/datum/preference/choiced/ui_style))
+	theme_cache[toshow] = ui_style
+
+	if (ui_style in storage_themes)
+		storage_themes[ui_style] += 1
```

Comment:
```suggestion
	if (!isnull(storage_themes[ui_style]))
		storage_themes[ui_style] += 1
```

Reply:
in is always slower then [] and implies an indexed form rather then an assoc one

Reply:
gotcha